test/valid_field_test.exs

```diff
@@ -23,6 +23,11 @@ defmodule ValidFieldTest do
       ValidField.with_changeset(%Model{})
       |> ValidField.assert_invalid_field(:title, ["", nil, "Something else"])
     end
+
+    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<2016-08-23>", fn ->
```

Comment:
@BlakeWilliams I meant something like:

`"Expected the following values to be invalid for 'date_of_birth': #Ecto.Date<2016-08-23>"